examples/keyed_list/src/main.rs

```diff
@@ -147,66 +154,95 @@ impl Component for Model {
         };
 
         html! {
-            <>
-                <div class="buttons">
-                    <button onclick=self.link.callback(|_| Msg::DeleteEverybody)>
-                        { "Delete everybody" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::CreatePersons(1))>
-                        { "Create 1" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::CreatePersons(5))>
-                        { "Create 5" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::CreatePersons(100))>
-                        { "Create 100" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::CreatePersons(500))>
-                        { "Create 500" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::CreatePersonsPrepend(1))>
-                        { "Prepend 1" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::CreatePersonsPrepend(5))>
-                        { "Prepend 5" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::SwapRandom)>
-                        { "Swap random" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::ReverseList)>
-                        { "Reverse list" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::SortById)>
-                        { "Sort by id" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::SortByName)>
-                        { "Sort by name" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::SortByAge)>
-                        { "Sort by age" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::SortByAddress)>
-                        { "Sort by address" }
-                    </button>
-                    <button onclick=self.link.callback(|_| Msg::ToggleKeyed)>
-                        { if self.keyed { "Disable keys" } else { "Enable keys" } }
+            <div class="container">
+            <p class="h2" ref=self.delta_ref.clone()/>
+            <hr />
+                <div class="row">
```

Comment:
The formatting here seems to have gotten messed up. Please properly format the contents of the `html!` macro.
This applies to the entire content in here.